### PR TITLE
Add Artemis sandbox configuration

### PR DESCRIPTION
**Pull request by dejanb**

### CODE REVIEW

File: broker/etc/artemis-broker-sandbox.xml [hunk 1]
```diff
@@ -0,0 +1,143 @@
...
+            <max-size-bytes>-1</max-size-bytes>
+            <message-counter-history-day-limit>10</message-counter-history-day-limit>
+            <address-full-policy>PAGE</address-full-policy>
+            <auto-create-queues>true</auto-create-queues>
+            <auto-create-addresses>true</auto-create-addresses>
+            <auto-create-jms-queues>true</auto-create-jms-queues>
+            <auto-create-jms-topics>true</auto-create-jms-topics>
+         </address-setting>
+         <!--default for catch all-->
+         <address-setting match="#">
```
Comment (sophokles73) [inline broker/etc/artemis-broker-sandbox.xml]: my understanding is that these settings apply to any (dynamically created) queue, e.g. to *event/tenantA* and *event/otherTenant*, right?

File: broker/etc/artemis-broker-sandbox.xml [hunk 2]
```diff
@@ -0,0 +1,143 @@
...
+            <dead-letter-address>DLQ</dead-letter-address>
+            <expiry-address>ExpiryQueue</expiry-address>
+            <redelivery-delay>0</redelivery-delay>
+            <!-- keep max size of the queue at 1MB -->
+            <max-size-bytes>1048576</max-size-bytes>
+            <!-- auto-expire messages after 5 minutes -->
+            <expiry-delay>300000</expiry-delay>
+            <message-counter-history-day-limit>10</message-counter-history-day-limit>
+            <address-full-policy>BLOCK</address-full-policy>
```
Comment (sophokles73) [inline broker/etc/artemis-broker-sandbox.xml]: this means that when the queue is full (1MB reached), the broker  grants no more credits upstream for the link representing the queue, right?

File: broker/etc/artemis-broker-sandbox.xml [hunk 3]
```diff
@@ -0,0 +1,143 @@
...
+         </address-setting>
+         <!--default for catch all-->
+         <address-setting match="#">
+            <dead-letter-address>DLQ</dead-letter-address>
+            <expiry-address>ExpiryQueue</expiry-address>
+            <redelivery-delay>0</redelivery-delay>
+            <!-- keep max size of the queue at 1MB -->
+            <max-size-bytes>1048576</max-size-bytes>
+            <!-- auto-expire messages after 5 minutes -->
+            <expiry-delay>300000</expiry-delay>
```
Comment (sophokles73) [inline broker/etc/artemis-broker-sandbox.xml]: a message is deleted from the queue after 5 minutes, right?

File: broker/etc/artemis-broker-sandbox.xml [hunk 4]
```diff
@@ -0,0 +1,143 @@
...
+            <auto-create-jms-topics>true</auto-create-jms-topics>
+         </address-setting>
+      </address-settings>
+
+       <wildcard-addresses>
+           <delimiter>/</delimiter>
+       </wildcard-addresses>
+
+   </core>
+</configuration>
```
Comment (sophokles73) [inline broker/etc/artemis-broker-sandbox.xml]: missing EOL

File: broker/etc/artemis-broker.xml [hunk 5]
```diff
@@ -133,14 +133,6 @@
          </address-setting>
       </address-settings>
```
Comment (sophokles73) [inline broker/etc/artemis-broker.xml]: queues are automatically created using these settings? When a device publishes to *event/BUMLUX* then the broker will automatically create a queue called *event/BUMLUX* and consumers can read events using a receiver link to the dispatch router with a source address of *event/BUMLUX*?

File: broker/etc/artemis-broker-sandbox.xml (hunk 1 above)
Comment (dejanb) [reply broker/etc/artemis-broker-sandbox.xml]: yes

File: broker/etc/artemis-broker-sandbox.xml (hunk 3 above)
Comment (dejanb) [reply broker/etc/artemis-broker-sandbox.xml]: yes

File: broker/etc/artemis-broker.xml (hunk 5 above)
Comment (dejanb) [reply broker/etc/artemis-broker.xml]: Yes. It should for our use case (and in demo mode we don't need more security). Integration tests are passing

**Comment by dejanb** [comment]
Moved the config file to the example/src/main/sandbox

**Pull request merged**